### ISSUE TITLE
feature request: `cargo-fuzz`

### ISSUE THREAD

**Issue by onkoe** [taiki-e/install-action#295]
it'd be nice to have :)

**Comment by taiki-e** [comment]
I would accept a PR to support this.

See [DEVELOPMENT.md](https://github.com/taiki-e/install-action/blob/main/DEVELOPMENT.md) for how to add support for a new tool.

(FWIW, this action maybe already supports cargo-fuzz via binstall fallback.)

**Comment by taiki-e** [comment]
> Do you think it's still worth adding it formally

Yeah, it would still be worth it because the officially supported tools have somehhow better installation performance and robustness.
